ccm_web/client/src/components/UserEnrollmentForm.tsx

```diff
@@ -215,6 +215,10 @@ export default function UserEnrollmentForm (props: UserEnrollmentFormProps): JSX
 
   const nameInput = (
     <>
+    {
+        showIncompleteAlerts && (firstName === undefined || lastName === undefined) &&
+          <InlineErrorAlert>You must fill in firstname or lastName.</InlineErrorAlert>
+    }
```

Comment:
I feel like this is already kinda handled by the field validation? If you try to hit enter with these, it should tell you that you're not allowed to leave it blank. I understand that's not the same pattern, but it does effectively do this.

Reply:
When you search for an email address and if it doesn't exists then First/Last Name component appear and at that point if user hits the Submit button, role and section section widget will have an error alert but not the first name and last name component. So I felt it was missed so i fixed it

Reply:
Okay, let me do some testing. I may suggest a slight change.

Reply:
As long as the first name and last name component is not displayed(in case of user already exists) the alert won't be appearing. So I feel this validation is still valid.

Reply:
Okay, I agree that that is a good addition/fix to the flow. Just need to correct the logic.

Reply:
You had an extra indentation, and I think I've improved the language.
```suggestion
    {
      showIncompleteAlerts && (firstName === undefined || lastName === undefined) &&
        <InlineErrorAlert>You must enter a first name and last name.</InlineErrorAlert>
    }
```